internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -250,25 +246,10 @@ func reconcileRoute(serviceName string) common.ReconcileFunc {
 	}
 }
 
-func isEnabled(request *common.Request) bool {
-	if request.Instance.GetAnnotations() == nil {
-		return false
-	}
-	if enable, isFound := request.Instance.GetAnnotations()[EnableAnnotation]; isFound {
-		if isEnabled, err := strconv.ParseBool(enable); err == nil {
-			return isEnabled
-		}
-	}
-	return false
-}
-
 func getVmConsoleProxyNamespace(request *common.Request) string {
 	const defaultNamespace = "kubevirt"
```

Comment:
What about setting the default namespace in a mutating webhook for the SSP CR and getting rid of this func?

Reply:
According to Andrej's comment [here](https://github.com/kubevirt/ssp-operator/pull/609#discussion_r1246259536), there is no default namespace anymore. So probably we can remove it.

Reply:
For now, this function can be left here. The namespace will be removed in a future PR, when the new version of Proxy is released.